frontend/src/index.js

```diff
@@ -22,9 +22,9 @@ ReactDOM.render(
           <LocalizationProvider dateAdapter={AdapterDateFns}>
             <SettingsProvider>
               <BrowserRouter>
-                <AuthProvider>
+                <LocalAuthProvider>
```

Comment:
Do we actually use it? It's weird that we have `LocalAuthProvider` in main `index.js`. It means to me that `CloudFront` is also using this

Reply:
You are correct, it should have been cognito auth. Fixed it